src/libraries/Common/src/System/Net/Http/Http2/Hpack/HPackEncoder.cs

```diff
@@ -503,6 +503,23 @@ public static byte[] EncodeLiteralHeaderFieldWithoutIndexingNewNameToAllocatedAr
             return span.Slice(0, length).ToArray();
         }
 
+        /// <summary>
+        /// Encodes a "Literal Header Field without Indexing - New Name" to a new array.
+        /// </summary>
+        public static byte[] EncodeLiteralHeaderFieldWithoutIndexingNewNameToAllocatedArray(string name, string value)
```

Comment:
Bit confused why this was added.

Reply:
Missed cleanup from Alt-Svc code; will remove.